manifests/params.pp

```diff
@@ -28,6 +28,7 @@
   $srv_domain          = $::domain
   # lint:ignore:puppet_url_without_modules
   $pluginsource        = 'puppet:///plugins'
+  $pluginfactsource    = 'puppet:///pluginfacts'
```

Comment:
Yeah, I'm OK with this, it'll go into a new minor version at least - people can override the parameter if necessary to get back to the old value.